packages/web3-core/package.json

```diff
@@ -2,16 +2,16 @@
 	"name": "web3-core",
 	"version": "4.0.1-alpha.3",
 	"description": "Web3 core tools for sub-packages. This is an internal package.",
-	"main": "dist/index.js",
+	"main": "lib/index.js",
 	"repository": "https://github.com/ChainSafe/web3.js",
 	"author": "ChainSafe Systems",
 	"license": "LGPL-3.0",
 	"files": [
-		"dist/**/*"
+		"lib/**/*"
```

Comment:
For npm whitelisting only `lib` , this will filter out `dist` form including in published lib. 
This will impact CDN as current CDN is configured to pick minified build through npm ( as per Greg info, I discussed with him last time ). 

I personally like it, as it will reduce lib size , so users node_modules will be of less size ( by excluding minified builds) but we will need to configure explicitly CDN for 4.x.

Reply:
Created #5743 to track this